src/libraries/System.Data.Common/System.Data.Common.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
```

Comment:
@ViktorHofer - the .sln files are still being updated automatically. I assume we don't want to check in these changes, correct?

Reply:
Right. Please undo this change.